CMakeLists.txt

```diff
@@ -586,7 +586,7 @@ if(testing)
 
     if(IS_DIRECTORY ${roottestdir})
       file(REMOVE_RECURSE ${CMAKE_CURRENT_BINARY_DIR}/roottest)
-      add_subdirectory(${roottestdir} roottest)
+      add_subdirectory(${roottestdir} roottest-build)
```

Comment:
Why we have here `roottest-build`?

Reply:
It's the *build* directory that's passed as the second argument. And I was always confused why I checked out `roottest` into my build dir ;-) I prefer to have the directory name clarify it's the roottest build dir. I'm okay to drop this if you find this wrong!

Reply:
nono, I was just curious...Let's use it!